app/browser/menu.js

```diff
@@ -693,11 +693,11 @@ const doAction = (action) => {
       }
       break
     case appConstants.APP_ON_CLEAR_BROWSING_DATA:
-      if (action.clearDataDetail.get('browserHistory')) {
-        appDispatcher.waitFor([appStore.dispatchToken], () => {
+      appDispatcher.waitFor([appStore.dispatchToken], () => {
+        if (appStore.getState().getIn(['clearBrowsingDataDefaults', 'browserHistory'])) {
```

Comment:
Because we are not sending anything in the action anymore. We have everything that we need directly in the store already. We are saving data here already https://github.com/brave/browser-laptop/pull/9457/files#diff-23ca389e2bcb77191b5a9c10900eb3a3R697